functions/__fisher_path_update.fish

```diff
@@ -5,7 +5,7 @@ function __fisher_path_update -a path
         git rebase --abort
         git fetch --quiet origin master
         git reset --quiet --hard FETCH_HEAD
-        git clean --quiet -d --force
+        git clean --quiet -dx --force
```

Comment:
-d is only for directories, so files added, but are ignored by .gitignore remain.  

  

-x removes ignored files

-X removes ignored files ONLY

> On Mar 15 2016, at 11:29 am, Jorge Bucaran &lt;notifications@github.com&gt;
wrote:  

>

> In [functions/__fisher_path_update.fish](https://github.com/fisherman/fisher
man/pull/157#discussion_r56111159):

>  
>  
>     > @@ -5,7 +5,7 @@ function __fisher_path_update -a path

>     >          git rebase --abort

>     >          git fetch --quiet origin master

>     >          git reset --quiet --hard FETCH_HEAD

>     > -        git clean --quiet -d --force

>     > +        git clean --quiet -dx --force

>  
>

> So, without this x untracked files will remain in the repo?

>

> What about -X?

>

> —  
You are receiving this because you authored the thread.  
Reply to this email directly or view it on GitHub:  
<https://github.com/fisherman/fisherman/pull/157/files#r56111159>![](https://g
ithub.com/notifications/beacon/ABlxkar8P6wnJKJJEoRvKmWkM5PoK2Ujks5ptieRgaJpZM4
HwrV6.gif)